java/cdk-environment/src/main/java/sleeper/environment/cdk/buildec2/BuildEC2Parameters.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sleeper.environment.cdk.buildec2;
+
+import sleeper.environment.cdk.config.AppContext;
+
+import java.util.Objects;
+
+import static sleeper.environment.cdk.config.AppParameters.*;
```

Comment:
Could be worth setting up an intellij codestyle in this repo so that we can define some rules such as not using wildcards in imports.

Reply:
We could take it as a separate issue to define that stuff and maybe check it statically? There are a lot of other places in the project where wildcard imports are used. This was done automatically by IntelliJ's default configuration. I have a save action set up to optimise imports.

Reply:
I think it's always good to catch these simple things as early as possible so they fit well with codestyles - I definitely prefer explicit imports so that you can see where a constant/.. has been imported from rather than guessing.

Reply:
I've changed my local code style configuration to stop it making wildcard imports, and removed this one.

Reply:
Here's an issue for code style configuration: https://github.com/gchq/sleeper/issues/15